### PR TITLE
[FIX] business requirement cost & sales price related issues

### PR DESCRIPTION
**Pull request by cialuo**
[IMP] Adjust view
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/xie8899/project/pull/25%23discussion_r55975890%22%2C%20%22https%3A//github.com/xie8899/project/pull/25%23discussion_r55976358%22%2C%20%22https%3A//github.com/xie8899/project/pull/25%23discussion_r56101610%22%2C%20%22https%3A//github.com/xie8899/project/pull/25%23issuecomment-196635025%22%2C%20%22https%3A//github.com/xie8899/project/pull/25%23issuecomment-196680470%22%2C%20%22https%3A//github.com/xie8899/project/pull/25%23issuecomment-197106611%22%2C%20%22https%3A//github.com/xie8899/project/pull/25%23issuecomment-197124731%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/25%23issuecomment-196635025%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40duanyp1991%20%40elicoidal%20%22%2C%20%22created_at%22%3A%20%222016-03-15T03%3A20%3A51Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7767792%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cialuo%22%7D%7D%2C%20%7B%22body%22%3A%20%22needs%20to%20be%20tested%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222016-03-15T06%3A11%3A47Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%2C%20%7B%22body%22%3A%20%22LGTM%2C%20%40elicoidal%20%20we%20first%20merge%20to%20have%20functional%20test%3F%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222016-03-16T01%3A50%3A43Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40duanyp1991%20Yes%2C%20merge%20and%20deploy%22%2C%20%22created_at%22%3A%20%222016-03-16T02%3A47%3A29Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20fc60e92796148f3c91b726f4266605c1235f134c%20business_requirement_deliverable/models/business.py%2028%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/25%23discussion_r55975890%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22The%20conversion%20code%20is%20not%20cool%20at%20all%21%20%3A%29%22%2C%20%22created_at%22%3A%20%222016-03-14T09%3A49%3A49Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%2C%20%7B%22body%22%3A%20%22Please%20do%20rewrite%20these%20code.%5Cr%5Cnyou%20can%20check%20the%20function%20of%20model%3A%20%60product.uom%60%3A%20%2A%2A_compute_qty%2A%2A.%22%2C%20%22created_at%22%3A%20%222016-03-14T09%3A53%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%2C%20%7B%22body%22%3A%20%22use%20standard%20function%20for%20that%20indeed.%20no%20need%20to%20reinvent%20the%20wheel%22%2C%20%22created_at%22%3A%20%222016-03-15T01%3A00%3A14Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20business_requirement_deliverable/models/business.py%3AL83-114%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull fc60e92796148f3c91b726f4266605c1235f134c business_requirement_deliverable/models/business.py 28'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/25#discussion_r55975890'>File: business_requirement_deliverable/models/business.py:L83-114</a></b>
- <a href='https://github.com/duanyp1991'></a> The conversion code is not cool at all! :)
- <a href='https://github.com/duanyp1991'></a> Please do rewrite these code.
  you can check the function of model: `product.uom`: **_compute_qty**.
- <a href='https://github.com/elicoidal'></a> use standard function for that indeed. no need to reinvent the wheel
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/25#issuecomment-196635025'>General Comment</a></b>
- <a href='https://github.com/cialuo'></a> @duanyp1991 @elicoidal
- <a href='https://github.com/elicoidal'></a> needs to be tested
- <a href='https://github.com/duanyp1991'></a> LGTM, @elicoidal  we first merge to have functional test?
- <a href='https://github.com/elicoidal'></a> @duanyp1991 Yes, merge and deploy

<a href='https://www.codereviewhub.com/xie8899/project/pull/25?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/xie8899/project/pull/25?approve=1'></a>&nbsp;<a href='https://github.com/xie8899/project/pull/25'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by xie8899** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/xie8899/project/pull/25'>Refresh</a>.<a href='#crh-update-none'></a>

File: business_requirement_deliverable/models/business.py [hunk 1]
```diff
@@ -80,11 +83,31 @@ def product_uom_change(self):
             unit_price = self.product_id.standard_price
         if self.uom_id and unit_price:
             if self.uom_id.uom_type == 'bigger':
-                self.unit_price = unit_price * self.uom_id.factor_inv
+                if self.product_id.uom_id.uom_type == 'bigger':
```
Comment (duanyp1991) [inline business_requirement_deliverable/models/business.py]: The conversion code is not cool at all! :)

Comment (duanyp1991) [inline business_requirement_deliverable/models/business.py]: Please do rewrite these code.
you can check the function of model: `product.uom`: **_compute_qty**.

Comment (elicoidal) [inline business_requirement_deliverable/models/business.py]: use standard function for that indeed. no need to reinvent the wheel

**Comment by cialuo** [comment]
@duanyp1991 @elicoidal

**Comment by elicoidal** [comment]
needs to be tested

**Comment by duanyp1991** [comment]
LGTM, @elicoidal  we first merge to have functional test?

**Comment by elicoidal** [comment]
@duanyp1991 Yes, merge and deploy

**Pull request merged**